components/raftstore/src/store/config.rs

```diff
@@ -495,11 +495,11 @@ impl Config {
         // Since the following configuration supports online update, in order to
         // prevent mistakenly inputting too large values, the max limit is made
         // according to the cpu quota.
```

Comment:
Could you clarify the factor `10` in the comment?

Reply:
Done